modules/etcd-cert-alerts-metrics-signer.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT    
+[id="etcd-cert-alerts-metrics-signer_{context}"]
+== etcd certificate rotation alerts and metrics signer certificates
+
+Two alert types inform users about pending `etcd` certificate expiration:
+[horizontal]
```

Comment:
Do you know for sure if our tooling handles this well (looks good on the preview, but I haven't seen it before so wonder about the Portal :sweat_smile:)

Reply:
@jeana-redhat I did a quick look at it, but couldn't find an example online. I decided that because it was documented in our doc references, that it would not be an issue. :-) I will make a note to verify this list after it hits the portal! Thanks for that input!

Reply:
Oooh.  I do not like that look at all! :grinning:

![image](https://github.com/openshift/openshift-docs/assets/25209155/393387f1-e2ae-4365-a5cf-96b992901b1e)

Reply:
Please validate that this looks ok on docs.redhat after GA.